docs/index.rst

```diff
@@ -43,80 +43,185 @@ to help you create fantastic documentation for your project.
 * **Importing your existing documentation**:
   :doc:`Import guide <intro/import-guide>`
 
+
 .. toctree::
    :maxdepth: 2
    :hidden:
-   :caption: First Steps
+   :caption: First steps
 
    intro/getting-started-with-sphinx
    intro/getting-started-with-mkdocs
+
    intro/import-guide
 
 
-.. _user-docs:
+Getting started with Read the Docs
+-----------------------------------
+
+Learn more about configuring your automated documentation builds
+and some of the core features of Read the Docs.
+
+* **Overview of core features**:
+  :doc:`features`
+
+* **Configure your documentation**:
+  :doc:`Configuration reference <config-file/index>` |
+  :doc:`webhooks` |
+  :doc:`badges` |
+  :doc:`Custom domains <custom_domains>`
+
+* **Connecting with GitHub, BitBucket, or GitLab**:
+  :doc:`Connecting your account <connected-accounts>`
+
+* **Read the Docs build and versioning process**:
+  :doc:`Build process <builds>` |
+  :doc:`Handling multiple docs versions <versions>`
+
+* **Troubleshooting**:
+  :doc:`support` |
+  :doc:`Frequently asked questions <faq>`
 
 .. toctree::
-   :maxdepth: 2
-   :caption: User Documentation
+   :maxdepth: 1
+   :hidden:
+   :caption: Getting started
 
-   versions
-   builds
    features
+
+   config-file/index
+   webhooks
+   badges
+   custom_domains
+
    connected-accounts
+
+   builds
+   versions
+
    support
    faq
-   config-file/index
-   guides/index
-   api/index
-   embed
 
-.. _feature-docs:
+
+Advanced features of Read the Docs
+----------------------------------
+
+Read the Docs offers many advanced features and options.
+Learn more about these integrations and how you can get the most
+out of your documentation and Read the Docs.
+
+* **Advanced project configuration**:
+  :doc:`subprojects` |
+  :doc:`Single version docs <single_version>` |
+  :doc:`Privacy levels <privacy>`
+
+* **Multi-language documentation**:
+  :doc:`Translations and localization <localization>`
+
+* **Redirects**:
+  :doc:`User defined redirects <user-defined-redirects>` |
+  :doc:`Automatic redirects <automatic-redirects>`
+
+* **Topic specific guides**:
+  :doc:`How-to guides <guides/index>`
+
+* **Extending Read the Docs**:
+  :doc:`REST API <api/index>` |
+  :doc:`embed`
 
 .. toctree::
    :maxdepth: 2
+   :hidden:
    :glob:
-   :caption: Feature Documentation
+   :caption: Advanced features
 
-   webhooks
-   badges
-   custom_domains
-   localization
-   vcs
    subprojects
-   conda
-   canonical
    single_version
    privacy
+
+   localization
+
    user-defined-redirects
    automatic-redirects
+
+   guides/index
+
+   api/index
+   embed
+
+   vcs
```

Comment:
These four sections (vcs, conda, etc.) seem to fit better in `docs/guides`. I propose moving them there. I also think `docs/features/embed` should be just a section of `docs/embed` rather than its own page.

Reply:
They might need a bit of a rewrite, but I agree the "Feature" documentation has always felt a bit weird, and should probably just be a list of features and then guides for how to use them well.